Schedule.API/Services/Procedures/RecommendationService.cs

```diff
@@ -253,5 +254,94 @@ private IEnumerable<EquipmentRelocationDto> FindCommonEquipmentRelocationAppoint
             }
             return equipmentRelocationDtos;
         }
+
+        public IEnumerable<RoomRenovationDto> RecommendRenovationAppointments(SchedulingDto dto)
+        {
+            if(dto.SecondRoomId == -1)
+                return BasicRenovationSearch(dto);
+            else
+                return ComplexRenovationSearch(dto);            
+        }
+
+        private IEnumerable<RoomRenovationDto> BasicRenovationSearch(SchedulingDto dto)
+        {
+            List<RoomRenovationDto> appointments = new List<RoomRenovationDto>();
+            List<Shift> availableShiftsSource = new List<Shift>();
+            IEnumerable<Shift> shiftsForSourceRoom = _shiftWrapper.Repository.GetShiftsByRoomID(dto.FirstRoomId);
+            foreach (Shift shift in shiftsForSourceRoom)
+            {
+                List<Shift> shiftList = new List<Shift>();
+                shiftList.Add(shift);
```

Comment:
This list will always have only one shift. Probably need to be defined before for.

Reply:
It should be like that, shiftList is used as parameter early created method, but I only need that one shift from shiftList.